package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "cf-docker-test-reporting",
-	"version": "1.0.7",
+	"version": "1.2.0",
```

Comment:
I was originally going to update the version to `2.0.0` but after reviewing the changes that have been made since the `test-report-link` branch merged, I realised that there weren't any breaking changes and most changes were either small features or bug fixes. I think a minor version bump is reasonable in this case.

Reply:
See my comment in the [Jira ticket](https://codefresh-io.atlassian.net/browse/CR-19339?focusedCommentId=66359) for a summary of the changes.